sdks/wasm/packager.cs

```diff
@@ -374,6 +391,7 @@ class WasmOptions {
 		AddFlag (p, new BoolFlag ("binding", "enable the binding engine", opts.AddBinding, b => opts.AddBinding = b));
 		AddFlag (p, new BoolFlag ("link-icalls", "link away unused icalls", opts.LinkIcalls, b => opts.LinkIcalls = b));
 		AddFlag (p, new BoolFlag ("il-strip", "strip IL code from assemblies in AOT mode", opts.ILStrip, b => opts.ILStrip = b));
+		AddFlag (p, new BoolFlag ("verbose", "set verbose option on link", opts.Verbose, b => opts.Verbose = b));
 
```

Comment:
This should be linker-version, or the comment should be changed to indicate it sets everything to verbose.

Reply:
done